### ISSUE TITLE
Popping page with animation returns to a blank white screen using a master detail (iOS only)

### ISSUE THREAD

**Issue by LeoJHarris** [AlexandrNikulin/AnimationNavigationPage#27]
Hi, not sure if this an issue with XF 2.4 but i strongly suspect this component that when i pop back to a page i am getting a blank white screen on iOS as it navigates back with animation.

I changed my pages to extend just ContentPage not AnimationPage (removed all references to animationnavigation) and the issue went away however i have lost my animation but i could return back to the correct page without the blank white screen when i pop the page.

So am calling this to pop:

`App.GetNavigation().Navigation.PopAsync(true);` 

In my App.cs:

```
public static AnimationNavigationPage GetNavigation()
        {
            RootPage root = Current.MainPage as RootPage; // root is the masterdetailpage
            return root?.CurrentNavigationPage;
        }
```

Am using xamairn forms 2.4.0.282

**Comment by LeoJHarris** [comment]
Any update regarding whether this is a confirmed bug or not and if so when we will likely see a bug fix released?

**Comment by AlexandrNikulin** [comment]
@LeoJHarris  Hi, sorry for delay.
Please tell me what version on control you use (latest is 1.5.6).
Also what animation you use and on what platform (iOS/Android). ?

**Comment by LeoJHarris** [comment]
I am using latest version 1.5.6 also i am using the push animation and issue only happens on ios.

**Comment by AlexandrNikulin** [comment]
@LeoJHarris Is it happens only with push animation ?
Try to do this:
1) Clean all -> Rebuild all
2) Try to use other animation type (for example Slide).
And please update me after it.

**Comment by LeoJHarris** [comment]
I havnt tested other animations just the push i will update you on those once i can test again and perhaps i will try to provide sample project if you require that?

**Comment by AlexandrNikulin** [comment]
@LeoJHarris  Nice! If you can - please send me sample project with this issue.

**Comment by LeoJHarris** [comment]
[Sample.zip](https://github.com/AlexandrNikulin/AnimationNavigationPage/files/1374098/Sample.zip)

See attached, issue is returning back to page when popping page contents are gone. Run on iOS EMULATOR

**Comment by AlexandrNikulin** [comment]
@LeoJHarris  I tested your sample proj and everything works fine.
Look: https://cl.ly/2Q3G2U3R0Y3L/Screen%20Recording%202017-10-11%20at%2009.29%20%D0%94%D0%9F.gif

Try "Clean all" - "Rebuild all".

**Comment by AlexandrNikulin** [comment]
@LeoJHarris Issue was on iOS less that 11 (for example 10.3).
Fixed in ver 1.5.7. If you need correct dll's now - please contact me
